guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>(1) A <s>Featue </s><strong> Feature </strong>Layer Object, usually refers to the data source&nbsp;</p><p>(2) hence a subset of Feature <s>Layer,,</s>  Layer, and in that case, when we use the distance <strong>feature,</strong> (Note: check extra comma and extra space before comma)&nbsp;</p><p>(3) Then the nearest_dist1, nearest_dist2 are <strong>the </strong>different distance from different features</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>